src/fhirdefs/FHIRDefinitions.ts

```diff
@@ -1,35 +1,23 @@
+import { flatten } from 'lodash';
+import {
+  cloneJsonMapValues,
+  addDefinitionToMap,
```

Comment:
I feel like this function is an internal implementation detail that should not be exported from `fhir-package-loader`.  I think we can rewrite the code in SUSHI so it does not use it and then eliminate this import (and FPL export).  See below for suggestion.

Reply:
Same as `cloneJsonMapValues`, I had already removed the export of this from FPL, so this was broken. Oops.

Removed the import in [1e1e8a9](https://github.com/FHIR/sushi/pull/1026/commits/1e1e8a9c1da76bf964a68f525572a28c3c562363).